crates/ethereum/src/state_update.rs

```diff
@@ -48,12 +49,7 @@ impl StateRootFetcher {
             Chain::Custom => EthereumBlockNumber(0),
         };
 
-        let signature = StateUpdateLog::signature();
-        let signature: ethers::types::H256 = signature.0.into();
-
-        let base_filter = Filter::default()
-            .address(vec![contract_address])
-            .topic0(signature);
```

Comment:
Can't we keep the `topic0` filtering with having an array of signatures we're interested in?

Something like:
```rust
.topic0(ValueOrArray::Array(vec![signature_old, signature])
```

Reply:
You're right. I always forget it takes an array.